meetings/2023-05/may-17.md

```diff
@@ -778,6 +776,7 @@ GB: We are following up on the feedback from the last session. We were originall
 GB: The first thing that was brought up at the beginning of the discussion was the question of the dynamic import JHD mentioned asymmetry from syntax. And RBN brought up, this stuff is difficult for a type system to support. And so we brought up the previously proposal that had been discussed that we all been fairly neutral on in the past and didn’t see new syntax and this is the idea of putting the phase into the – into the dynamic import statement itself. It becomes what we are calling a metamethod. It’s not a metaproperty. But it’s part of the import call form.
 
 ```js
+
```

Comment:
😭

Reply:
woof

Reply:
savaged again by the emotional support whitespace!  🤣

Reply:
hang on, I actually think I can address this via some additional negative lookbehind that may not result in the regex being too fragile...

Reply:
no quantifier support for neg lookbehind, so fix is a hack